doc/tutorials/nvmx_virtualization.rst

```diff
@@ -0,0 +1,169 @@
+.. _nested_virt:
+
+Enable Nested Virtualization
+############################
+
+With nested virtualization enabled, it's possible to run virtual machine
+instances inside of level 1 guests. Though it seems both nested guests and
+User VMs are launched from the guest VM, the distinction between them is
+that, in the case of User VM, hypervisor is not running on Service OS, and
+besides the device model, ACRN provides all the support to run the User VMs.
+While for nested guests, an instance of guest hypervisor is running on the
+guest to manage the nested VMs.
+
```

Comment:
a little confused about the name "User VMs", "guest VM" and "nested guests", but I get your mean.

Reply:
Yes, it's confusing. I'm trying to differentiate them:

-  User VM is the 'official' name for UOS, which is launched by acrn-dm;
-  Nested guest, or L2 guest, L2 VM: the virtual machine launched by L1 hypervisor, and runs on top of the L1 guest;
-  Guest VM: is a generic/loose term refers to the virtual machine runs on top of a hypervisor. It may refer to any of the above VMs, depending on the context;